addons/mail/static/src/components/chatter/chatter_suggested_recipient_tests.js

```diff
@@ -374,6 +374,34 @@ QUnit.test("suggested recipients list display 3 suggested recipient and 'show mo
     );
 });
 
+QUnit.test("no recipient on 'Log note' composer", async function (assert) {
+    assert.expect(1);
+
+    this.data['res.partner'].records.push({
+        display_name: "John Jane",
+        email: "john@jane.be",
+        id: 100,
+    });
+    this.data['res.fake'].records.push({
+        id: 10,
+        partner_ids: [100],
+    });
+    await this.start();
+    const chatter = this.env.models['mail.chatter'].create({
+        threadId: 10,
+        threadModel: 'res.fake',
+    });
+    await this.createChatterComponent({ chatter });
+    await afterNextRender(() =>
+        document.querySelector(`.o_ChatterTopbar_buttonLogNote`).click()
+    );
+    assert.containsNone(
+        document.body,
+        '.o_ComposerSuggestedRecipientList',
+        "Should not contain recipients when opening the composer from 'log note' button"
+    );
```

Comment:
The goal here is not to test the interface (which was already correct before the mentioned PR), but to test that the RPC done when sending the note does not contain the recipients as param.